### PR TITLE
Use core collection creation policies

### PR DESCRIPTION
**Pull request by zachmullen**
This depends on a forthcoming branch in girder/girder

### CODE REVIEW

**Comment by jeffbaumes** [comment]
Nice, but the module creation also makes the Data, Analysis, and Visualization folders in the collection - don't we need to add that explicitly here now? See https://github.com/Kitware/romanesco/pull/102/files#diff-611548d119cee7cc9600b3275d888a81L132

**Comment by zachmullen** [comment]
Good point, forgot about that. I will add that code also.

File: app/src/app.js [hunk 1]
```diff
@@ -272,21 +272,24 @@
...
                     $('.new-collection-name').val('');
                     this.collection.fetch({}, true);
-                }, this)).error(_.bind(function (xhr, status, message) {
+
+                    _.each(['Data', 'Analyses', 'Visualizations'], function (name) {
+                        new girder.models.FolderModel({
+                            parentType: 'collection',
+                            parentId: collection.get('_id'),
+                            name: name
+                        }).save();
```
Comment (jeffbaumes) [inline app/src/app.js]: @zachmullen the folders need to be public based on isPublic also. Right now they are always private, even if the collection is public.

Comment (zachmullen) [inline app/src/app.js]: Good catch, forgot that parameter :)

**Comment by zachmullen** [comment]
@jeffbaumes should be fixed now

**Comment by jeffbaumes** [comment]
LGTM along with https://github.com/Kitware/romanesco/pull/102

**Pull request merged**